ouroboros-consensus/src/consensus-testlib/Test/Util/LedgerStateOnlyTables.hs

```diff
@@ -12,23 +13,36 @@
 -- This is useful when we only need a ledger state and ledger tables, but not
 -- necessarily blocks with payloads (such as defined in @Test.Util.TestBlock@).
 module Test.Util.LedgerStateOnlyTables (
-    LedgerTables (..)
-  , OTLedgerState (..)
+    OTLedgerState
+  , OTLedgerTables
+  , pattern OTLedgerState
+  , pattern OTLedgerTables
   ) where
 
 import           Cardano.Binary (FromCBOR (..), ToCBOR (..))
 import           GHC.Generics (Generic)
 import           NoThunks.Class (NoThunks)
+import           Ouroboros.Consensus.Ledger.Basics (LedgerState)
 import           Ouroboros.Consensus.Ledger.Tables
-import           Ouroboros.Consensus.Ledger.Tables.Utils
+                     (CanSerializeLedgerTables (..), CanStowLedgerTables (..),
+                     CodecMK (..), HasLedgerTables (..), MapKind, NameMK (..),
+                     ValuesMK)
+import           Ouroboros.Consensus.Ledger.Tables.Utils (emptyLedgerTables)
 
 {-------------------------------------------------------------------------------
   Simple ledger state
 -------------------------------------------------------------------------------}
 
-data OTLedgerState k v (mk :: MapKind) = OTLedgerState {
+type OTLedgerState  k v = LedgerState  (OTBlock k v)
+type OTLedgerTables k v = LedgerTables (OTLedgerState k v)
+
+-- | An empty type for blocks, which is only used to record the types @k@ and
+-- @v@.
+data OTBlock k v
+
+data instance LedgerState (OTBlock k v) (mk :: MapKind) = OTLedgerState {
     otlsLedgerState  :: ValuesMK k v
-  , otlsLedgerTables :: LedgerTables (OTLedgerState k v) mk
+  , otlsLedgerTables :: OTLedgerTables k v mk
   }
```

Comment:
INFO: We were not actually defining the `LedgerState` as a `data instance`, which this change fixes. Furthermore, since we don't want to mention the block type (it is empty anyway), we provide the `OTLedgerState` and `OTLedgerTables` type synonyms. The `OTBlock` type isn't exported.